docs/index.md

```diff
@@ -4,6 +4,9 @@ description: Welcome to React Native Firebase! To get started, you must first se
 next: /typescript
 ---
 
+> React Native Firebase has begun to deprecate the namespaced API (i.e firebase-js-sdk `< v9` chaining API). React Native Firebase will be moving to the modular API (i.e. firebase-js-sdk `>= v9`) in the next major release. See [migration guide]() for more information.
```

Comment:
everything looks beautiful except the guide isn't hooked up to the URL here, you probably know that
only commenting to say this all looks like a +1 from me otherwise, even though it's draft at moment